dist/react-json-schema.js

```diff
@@ -34,7 +34,7 @@ var ReactJsonSchema = function () {
   }, {
     key: 'parseSubSchemas',
     value: function parseSubSchemas() {
-      var subSchemas = arguments.length <= 0 || arguments[0] === undefined ? [] : arguments[0];
+      var subSchemas = arguments.length > 0 && arguments[0] !== undefined ? arguments[0] : [];
```

Comment:
Yeah, compiler probably updated to && for better performance (doesn't perform right hand evaluation if left hand fails, unlike ||).